client-web/src/exlibris-app/exlibris-app.html

```diff
@@ -98,12 +102,40 @@
                         attr-for-selected="data-page"
                         fallback-selection="view404"
                         role="main">
+                    <exlibris-book-grid data-page="all-books"
+                                        books="[[allBooks]]"
+                                        message="[[bookMessage.ALL]]"
+                                        on-exlibris-book-grid-lower-threshold="_handleLowerThreshold"></exlibris-book-grid>
+                    <exlibris-book-grid data-page="overdue-books"
+                                        books="[[overdueBooks]]"
+                                        message="[[bookMessage.OVERDUE]]"></exlibris-book-grid>
+                    <exlibris-book-grid data-page="borrowed-books"
+                                        books="[[borrowedBooks]]"
+                                        message="[[bookMessage.BORROWED]]"></exlibris-book-grid>
+                    <exlibris-book-grid data-page="reserved-books"
+                                        books="[[reservedBooks]]"
+                                        message="[[bookMessage.RESERVED]]"></exlibris-book-grid>
+                    <exlibris-book-grid data-page="expected-soon"
+                                        books="[[expectedSoonBooks]]"
+                                        message="[[bookMessage.EXPECTED_SOON]]"></exlibris-book-grid>
+                    <exlibris-book-grid data-page="should-return-books"
+                                        books="[[shouldReturnBooks]]"
+                                        message="[[bookMessage.SHOULD_RETURN_SOON]]"></exlibris-book-grid>
                     <div data-page="view404">View404</div>
                 </iron-pages>
             </app-header-layout>
         </app-drawer-layout>
     </template>
     <script>
+        const EmptyArrayMessage = {
```

Comment:
`EmptyArrayMessage ` is set to global context. Why does ESLint ignore this?

Reply:
@dkalinin  Where to put it?

Reply:
Use {} to create a separate scope: http://es6-features.org/#BlockScopedFunctions